core/src/test/java/org/mobilitydata/gtfsvalidator/parsing/CsvRowTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.parsing;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.Reader;
+import java.io.StringReader;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@RunWith(JUnit4.class)
+public class CsvRowTest {
+
+    @Test
+    public void shouldReturnRowNumber() {
+        Reader reader = new StringReader("stop_id,stop_name,stop_lat" + System.lineSeparator() +
+                "s1,First stop,3.21" + System.lineSeparator() + "s2,Second stop,1.31" + System.lineSeparator());
+        CsvFile csvFile = new CsvFile(reader, "stops.txt");
+
+        CsvRow underTest = new CsvRow(csvFile, 4, new String[]{});
```

Comment:
You setup a file with 3 rows of data but you pass `rowNumber = 4` and arbitrary columnValues to the constructor of `CsvRow`. This is may confuse the reader.

I understand that you may want to test CsvRow without actually invoking the CSV parser. Would you like to pass realistic data to `CsvRow()` constructor?

You can also test all the methods of CsvRow in a single test because they are short, fast and trivial.